src/Rapidez.php

```diff
@@ -130,6 +130,13 @@ public function setStore(Store|array|callable|int|string $store): void
         config()->set('rapidez.root_category_id', $store['root_category_id']);
         config()->set('frontend.base_url', url('/'));
 
+        // Set all store-specific values
+        foreach (array_keys(config('rapidez')) as $config) {
+            foreach (config('rapidez.stores.' . $store['code'] . '.' . $config, []) as $key => $value) {
+                config()->set('rapidez.' . $config . '.' . $key, $value);
+            }
+        }
+
```

Comment:
When changing to multiple stores (like we do during indexing) wouldn't we start setting config from store 2 on store 1, instead of setting the config from store 2 on the default config.

If that's the case we might not actually get the default config if we haven't overridden it in store 2, instead getting the config that store 1 overrode.

Reply:
Good points---I've now added a thing that sets "default" values when they get changed, and then uses that to reset to when switching store again.